modules/port-connection/main.tf

```diff
@@ -154,7 +154,7 @@ resource "equinix_fabric_connection" "secondary_port_connection" {
       }
       link_protocol {
         type       = one(data.equinix_fabric_ports.aside_secondary_port[0].data.0.encapsulation).type
-        vlan_tag   = one(data.equinix_fabric_ports.aside_secondary_port[0].data.0.encapsulation).type == "DOT1Q" ? var.aside_vlan_tag : null
+        vlan_tag   = one(data.equinix_fabric_ports.aside_secondary_port[0].data.0.encapsulation).type == "DOT1Q" ? var.secondary_aside_vlan_tag : null
```

Comment:
@thejptnaz , Great idea to have them be able to be separated. Please add the additional change for QINQ as well. Then we can approve and get it merged and released.